.pylintrc

```diff
@@ -1,10 +1,16 @@
 [BASIC]
-
-# Regular expression matching correct function names
-function-rgx=[a-z_][a-z0-9_]{2,50}$
+# In order to make a check more strict add proper regex http://pylint-messages.wikidot.com/messages:c0103
+argument-rgx=.*
+attr-rgx=.*
+class-rgx=.*
+const-rgx=.*
+function-rgx=.*
+method-rgx=.*
+module-rgx=.*
+variable-rgx=.*
+inlinevar-rgx=.*
```

Comment:
Doesn't this mean we can name functions, etc. whatever we want? Matching as anything doesn't make much sense to me

Reply:
Just following Leapp guide lines, but we can go back, what you think about it @bocekm ?

Reply:
I think these benevolent regexps are not needed. We've already fixed the annoying one https://github.com/oamg/convert2rhel/pull/53, the rest seems to be fine. We can discuss particular ones if necessary instead of "disabling" all.
By the way, @sturivny has resolved the invalid name "violations" here https://github.com/oamg/convert2rhel/pull/54 already.